Makefile

```diff
@@ -47,16 +47,20 @@ dev: database ethprovider hub wallet proxy
 prod: database-prod hub-prod proxy-prod
 
 stop: 
-	docker container stop $(project)_buidler 2> /dev/null || true
 	bash ops/stop.sh
+	docker container stop $(project)_buidler 2> /dev/null || true
 	docker container prune -f
 
 reset: stop
-	docker volume rm connext_chain_dev || true
-	rm -f $(contracts)/build/state-hash
-	docker volume rm connext_database_dev || true
+	docker volume rm connext_database_dev connext_chain_dev || true
 	docker volume rm `docker volume ls -q | grep "[0-9a-f]\{64\}" | tr '\n' ' '` 2> /dev/null || true
 
+reset-client:
```

Comment:
We don't need to do this every time we build. For now, keep it available as a separate rule.